tests/main/xdg-settings/task.yaml

```diff
@@ -61,11 +61,20 @@ execute: |
         export DBUS_SESSION_BUS_ADDRESS=$DBUS_SESSION_BUS_ADDRESS
 
         # run xdg-settings from inside the snap
-        test-snapd-xdg-settings.set-default-web-browser browser.desktop
+        test-snapd-xdg-settings.xdg-settings-wrapper "$@"
+
+        if [[ $# -eq 3 ]]
+        then
+            # rewrite final param to include snap name
```

Comment:
I didn't follow how this related to merging from master, but you just meant to update the comment here right?

Reply:
I was just saying if you are doing other commits, which might include a merge from master, you could fix this nitpick. If everything was all green, it would be ok to skip. You fixed it though. Thanks!